src/API/Site/Controllers/TourController.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+	declare(strict_types=1);
+
+	namespace Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers;
+
+	use Automattic\WooCommerce\GoogleListingsAndAds\API\TransportMethods;
+	use Automattic\WooCommerce\GoogleListingsAndAds\Options\OptionsInterface;
+	use Automattic\WooCommerce\GoogleListingsAndAds\Proxies\RESTServer;
+	use WP_REST_Request as Request;
+	use WP_REST_Response as Response;
+	use Exception;
+
+	defined( 'ABSPATH' ) || exit;
+
+	/**
+	 * Class for handling API requests for getting and update the tour visualizations.
+	 *
+	 * @package Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers
+	 */
+class TourController extends BaseOptionsController {
+
+
+	/**
+	 * Constructor.
+	 *
+	 * @param RESTServer $server
+	 */
+	public function __construct( RESTServer $server ) {
+		parent::__construct( $server );
+	}
+
+	/**
+	 * Register rest routes with WordPress.
+	 */
+	public function register_routes(): void {
+		/**
+		 * GET The tour visualizations
+		 */
+		$this->register_route(
+			'/tours/(?P<id>[a-z]+)',
```

Comment:
Interesting, they hanks. I ended up adding A-Z in the regex. Even the a-z supports them as you mentioned  I think it will be clear to explicitly add it in the regex.